kura/examples/org.eclipse.kura.example.eddystone/OSGI-INF/metatype/org.eclipse.kura.example.eddystone.EddystoneExample.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2011, 2016 Eurotech and/or its affiliates
+
+     All rights reserved. This program and the accompanying materials
+     are made available under the terms of the Eclipse Public License v1.0
+     which accompanies this distribution, and is available at
+     http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Eurotech
+
+-->
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.example.eddystone.EddystoneExample"
+         name="Eddystone Example" 
```

Comment:
We could rename our examples in this way:
IBeaconExample -> IBeaconAdvertiser
IBeaconScannerExample -> IBeaconScanner
EddystoneExample -> EddystoneAdvertiser
EddystoneScannerExample -> EddystoneScanner